src/main/common/helpers/courtOrderHelper.ts

```diff
@@ -0,0 +1,35 @@
+import { Claim } from 'claims/models/claim'
+
+import { DraftClaimantResponse } from 'features/claimant-response/draft/draftClaimantResponse'
+import { ResponseDraft } from 'features/response/draft/responseDraft'
+
+import { FullAdmissionResponse } from 'claims/models/response/fullAdmissionResponse'
+import { PartialAdmissionResponse } from 'claims/models/response/partialAdmissionResponse'
+
+import { PaymentPlanHelper } from 'shared/helpers/paymentPlanHelper'
+import { PaymentPlan } from 'common/payment-plan/paymentPlan'
+import { Frequency } from 'common/frequency/frequency'
+import { CourtOrder } from 'common/court-order/courtOrder'
+import { StatementOfMeansCalculations } from 'common/statement-of-means/statementOfMeansCalculations'
+
+export class CourtOrderHelper {
```

Comment:
As above, the name needs to be more specific as we will have more court orders in the future